tools/donate_cpu_lib.py

```diff
@@ -133,7 +133,10 @@ def checkout_cppcheck_version(repo_path, version, cppcheck_path):
         hash_old = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD'], cwd=cppcheck_path).strip()
 
         print('Pulling {}'.format(version))
-        subprocess.check_call(['git', 'pull'], cwd=cppcheck_path)
+        # --rebase is a workaround for a dropped commit - see https://github.com/danmar/cppcheck/pull/6904
```

Comment:
This does not help for 6904. But it can help in the future if we need to something like this again.

Reply:
It strikes me now that it's really unfortunate that we didn't merge this before I dropped the commit. :-(

Reply:
> It strikes me now that it's really unfortunate that we didn't merge this before I dropped the commit. :-(

ah.. that wouldn't have helped at all.